lib/activerecord-import/adapters/mysql_adapter.rb

```diff
@@ -32,21 +34,26 @@ def insert_many( sql, values, *args ) # :nodoc:
 
     # if we can insert it all as one statement
     if NO_MAX_PACKET == max || total_bytes < max
-      number_of_inserts += 1
+      number_of_inserts = values.size
       sql2insert = base_sql + values.join( ',' ) + post_sql
-      insert( sql2insert, *args )
+      first_inserted_id = insert( sql2insert, *args )
     else
       value_sets = ::ActiveRecord::Import::ValueSetsBytesParser.parse(values,
         reserved_bytes: sql_size,
         max_bytes: max)
       value_sets.each do |value_set|
         number_of_inserts += 1
         sql2insert = base_sql + value_set.join( ',' ) + post_sql
-        insert( sql2insert, *args )
+        ids << insert( sql2insert, *args )
       end
     end
 
-    [number_of_inserts, []]
+    if first_inserted_id
+      last_inserted_id = first_inserted_id + number_of_inserts - 1
+      ids = (first_inserted_id..last_inserted_id).to_a
```

Comment:
@robertomiranda, thank you for contributing to ar-import! With regard to your question above I have the same concerns as @tonic20 and @jkowens. I'd want to ensure that we however this implemented we can guarantee reliability and avoid unsuspecting issues when the returned IDs are wrong.